lib/components/form/advanced-settings-panel.tsx

```diff
@@ -127,7 +128,7 @@ const AdvancedSettingsPanel = ({
   modeSettingDefinitions: ModeSetting[]
   modeSettingValues: ModeSettingValues
   onPlanTripClick: () => void
-  setQueryParam: (evt: any) => void
+  setQueryParam: (evt: QueryParamChangeEvent) => void
```

Comment:
Don't use this type here, in other places in OTP RR I believe we are just using a simple `(evt: any) => void`. If you want to make a new type and implement it for all the setQueryParams that would be awesome, but we shouldn't import it from OTP UI.